cmd/crc/cmd/console.go

```diff
@@ -52,9 +55,19 @@ func runConsole(arguments []string) {
 	}
 
 	if consoleURLMode {
-		output.Out(result.URL)
+		output.Out(result.ClusterConfig.WebConsoleURL)
+	}
+	if consolePrintCredentials {
+		output.Out("To login as a normal user, username is 'developer' and password is 'developer'.")
+		output.OutF("To login as an admin, username is 'kubeadmin' and password is '%s'.", result.ClusterConfig.KubeAdminPass)
+	}
+	if consoleURLMode || consolePrintCredentials {
 		return
 	}
+
+	if result.State != state.Running {
+		errors.ExitWithMessage(1, "CodeReady Containers instance is not running, cannot open the OpenShift Web console")
+	}
 	output.Out("Opening the OpenShift Web console in the default browser...")
-	_ = browser.OpenURL(result.URL)
+	_ = browser.OpenURL(result.ClusterConfig.WebConsoleURL)
```

Comment:
ignore the error?

Reply:
This was ignored before my changes. What should we do when this fails? Print a "Failed to open web console, you can access it by opening %s in your web browser"

Reply:
I know it was ignored, and perhaps not a good idea. The suggested output sounds good, as in that case at least the user has an option to continue.